vars/piperExecuteBin.groovy

```diff
@@ -48,6 +48,9 @@ void call(Map parameters = [:], String stepName, String metadataFile, List crede
                 echo "Context Config: ${config}"
             }
 
+            //Add ANS credential information to the config
+            config += ["ansServiceKeyCredentialsId": script.commonPipelineEnvironment.configuration.hooks.ansServiceKeyCredentialsId]
```

Comment:
There are two ans related things the user might configure:
1. service key
2. event template file path

Let's define them in a sub-field like so:
```
hooks:
  ans:
    serviceKeyCredentialsIs: 'my-creds'
    eventTemplateFilePath: './my/file/path'
```
So the code should be:
```suggestion
            config += ["ansServiceKeyCredentialsId": script.commonPipelineEnvironment.configuration.hooks?.ans?.serviceKeyCredentialsId]
```